brave/brave-core
# Use panel for Rewards page onboarding

Resolves https://github.com/brave/brave-browser/issues/24538

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Open the browser with a new profile.
- Navigate to `brave://rewards`.
- Click the `Start Using Rewards` button.
- Verify that the Rewards panel is opened with onboarding displayed.
- Click "Start Using Rewards" on the panel.
- Close the panel.
- Verify that Rewards page content is displayed.
